servicetalk-http-api/src/main/java/io/servicetalk/http/api/ClearAsyncContextHttpServiceFilter.java

```diff
@@ -21,10 +21,10 @@
 /**
  * Clears the {@link AsyncContext} for a new request.
  */
-final class ClearAsyncContextHttpServiceFilter implements
+public final class ClearAsyncContextHttpServiceFilter implements
```

Comment:
Discussed offline: this class doesn't have to be public, it can move to the `http-netty` module as pkg-private

Reply:
We should at some point consider an `http-internal` module for the non-Netty specific HTTP internals.

Reply:
The new module doesn't make much sense if classes from that module are not used by other 2+ modules. For `http-netty` only, it's more restrictive to use pkg-private classes.
There are a few other utilities that will be nice to share between `http-api` and `http-netty`, but there is "chicken and egg" problem as those utilities depend on `http-api` 😕